docs/builtins.md

```diff
@@ -1256,3 +1256,463 @@ x-axis and the ray from the origin to the point `x y`. More formally,
+
+The `line` function draws a line from the current cursor position to the
+given `x` and `y` coordinates. The cursor position is then updated to
+the given coordinates, which allows for easy polygon drawing.
+
+### `rect`
+
+`rect` draws a rectangle with the given width and height at the pen's
+current position.
+
+#### Example
+
+```evy
+grid
+move 40 20
+rect 10 30
+rect 40 20
+```
+
+Output
+
+<img width="300" alt="two black rectangles with touching vertices" src="img/rects.png">
+
+#### Reference
+
+    rect width:num height:num
+
+The `rect` function draws a rectangle with the given `width` and
+`height` at the current cursor position. The cursor position is then
+updated to the previous position plus the given `width` and `height`.
+
+### `circle`
+
+`circle` draws a circle with given radius at the pen's current position.
+
+#### Example
+
+```evy
+grid
+move 50 50
+circle 10
+```
+
+Output
+
+<img width="300" alt="black circle in the canvas center" src="img/circle.png">
+
+#### Reference
+
+    circle radius:num
+
+The `circle` function draws a circle with the given `radius` centered
+at the current cursor position. The cursor position does not change
+after drawing a circle.
+
+### `width`
+
+`width` sets the thickness of the lines drawn by the pen.
+
+#### Example
+
+```evy
+width 10
+line 30 30
+width 1
+line 60 60
+width 0.1
+line 90 90
+grid
+```
+
+Output
+
+<img width="300" alt="diagonal lines of different width" src="img/width.png">
+
+#### Reference
+
+    width n:num
+
+The `width` function sets the thickness of the _stroke_ to the given `n`
+units. The stroke is the visible line that is drawn when using the
+`line` function or any other shape function after setting
+`fill "none"`.
+
+### `color`
+
+`color` changes the color of the drawing pen. All
+[CSS (Cascading StyleSheets) color values](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value) are
+supported. You can start by using the simpler
+[named CSS colors](https://developer.mozilla.org/en-US/docs/Web/CSS/named-color),
+such as `"red"`, `"darkmagenta"`, and `"lightseagreen"`.
+
+#### Example
+
+```evy
+color "darkmagenta"
+rect 20 20
+```
+
+Output
+
+<img width="300" alt="dark magenta square in bottom left corner" src="img/magenta.png">
+
+#### Reference
+
+    color c:string
+
+The `color` function changes the color of the _stroke_ and the _fill_ to
+the given CSS color string `c`. Evy supports all
+[CSS color values](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value),
+including semi-transparent ones. For example, the following code changes
+the color to a shade of red that is 60% opaque:
+`color "hsl(0deg 100% 50% / 60%)"`.
+
+_named CSS colors_, such as `"red"`, `"darkmagenta"`, and
+`"lightseagreen"`, are a simpler way of specifying
+common colors. For a complete list of named CSS colors, see the
+[Mozilla Developer documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/named-color).
+
+### `colour`
+
+See [`color`](#color).
```

Comment:
Text: `colour` is an alternate spelling of `color`. See [`color`](#color).

Reply:
thanks, done.